scripts/generate-protos.sh

```diff
@@ -18,16 +19,32 @@ rm -rf "${TSOUTDIR}"
 mkdir -p "${TSOUTDIR}"
 cp ${TSBASEDIR}/packagefiles/* "${TSOUTDIR}"
 
+rm -rf "${RUBYDIR}"
+mkdir -p "${RUBYDIR}"
+
+
 pushd "${PROTOPATH}"
 FILES=$(find . -type f -name "*.proto")
 echo $FILES
 
+bundle install
+
 for proto in $FILES; do
     protoc \
-        --go_out="${GOOUTDIR}" --go_opt=module="github.com/speakeasy-api/speakeasy-schemas/grpc/go" --go-grpc_out="${GOOUTDIR}" --go-grpc_opt=module="github.com/speakeasy-api/speakeasy-schemas/grpc/go" \
```

Comment:
Sorry about the whitespace changes.  I can revert if they are distracting, I changed it for readability.

Reply:
that should be fine, we don't have a linter action on this repo